build/prepare/CheckPrereqs.targets

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<Import Project="UbuntuCoreclrAndCoreFxDependencies.props" />
+
+	<Target Name="CheckPrereqs"
+			DependsOnTargets="CheckUbuntuAptGetDependencies;
+												CheckCentOsYumDependencies;">
+
+		<Message Text="Using Stage0 version:" />
+		<Exec Command="$(DotnetStage0) --version" />
+	</Target>
+
+	<Target Name="CheckUbuntuAptGetDependencies"
+			Condition=" '$(Rid)' == 'ubuntu.14.04-x64' ">
+
+		<Exec Command="dpkg -s %(UbuntuCoreclrAndCoreFxDependencies)" />
+
+		<Exec Command="dpkg -s %(DebianPackageBuildDependencies)" />
+
+		<OnError ExecuteTargets="ShowAptGetUsage" />
```

Comment:
What if we were to do target batching here instead of just task batching, and capture the package currently being checked into a property so that `ShowAptGetUsage` will know which package to print wasn't installed?